smrf/framework/recipes.ini

```diff
@@ -69,6 +69,20 @@ trigger_section:
 precip:
   apply_defaults = True
 
+[precip_input_scalar_recipe]
```

Comment:
I've been trying to test recipes as well to ensure that they are working. I have only implemented it for wind but it's something we're tying to build out since there are some odd things that can happen with the recipes and we want to make sure that we're not getting any extra outputs that we don't need.

Reply:
Sounds good. I'll take a look and see what I can do

Reply:
The recipe tests are fairly straight forward, there is an example in there already to follow but this will be a lot simpler.

Reply:
This should be addressed in the latest round of commits.